reactive/kubernetes_worker.py

```diff
@@ -657,6 +617,13 @@ def configure_kubelet(dns, ingress_ip):
     kubelet_opts['logtostderr'] = 'true'
     kubelet_opts['node-ip'] = ingress_ip
 
+    endpoint_config = \
+        endpoint_from_flag('endpoint.container-runtime.available').get_config()
```

Comment:
Can we ever call create_config without the runtime available?

Reply:
I don't think so; `configure_kubelet` is only called by `start_worker`, which requires `endpoint.container-runtime.available` to be set.

Reply:
works for me. I just wanted to make sure we thought about it.